packages/tupaia-web/src/features/Dashboard/DashboardMenu.tsx

```diff
@@ -33,7 +33,9 @@ const ItemButton = styled(Menu)`
     background: ${({ theme }) => theme.palette.background.default};
   }
   .MuiMenu-paper {
-    max-height: calc(100vh - ${TOP_BAR_HEIGHT});
+    max-height: calc(
+      100vh - ${TOP_BAR_HEIGHT} + ${TOP_BAR_HEIGHT}
```

Comment:
I assume that you tested this and it works but I'm surprised calc works this way. Shouldn't it be either `100vh - ${TOP_BAR_HEIGHT} - ${TOP_BAR_HEIGHT}` or `100vh - (${TOP_BAR_HEIGHT} + ${TOP_BAR_HEIGHT})`?